src/sinks/util/encoding/encoding_config.rs

```diff
@@ -0,0 +1,142 @@
+use crate::sinks::util::encoding::{
+    encoding_config_with_default::EncodingConfigWithDefault, inner::Inner, EncodingConfiguration,
+    TimestampFormat,
+};
+use serde::de::{DeserializeOwned, IntoDeserializer, MapAccess, Visitor};
+use serde::{de, Deserialize, Deserializer, Serialize};
+use std::fmt;
+use std::fmt::Debug;
```

Comment:
Just a nit, but these can be combined: `use std::fmt::{self, Debug};` So could the `serde` lines, but that's will end up making it a larger block and is something we've done both ways. It would be nice if `rustfmt` would standardize that for us.

Reply:
You're right!